uvicorn/supervisors/multiprocess.py

```diff
@@ -56,8 +56,11 @@ def startup(self) -> None:
             signal.signal(sig, self.signal_handler)
 
         for _idx in range(self.config.workers):
+            logger.debug(f"Getting subprocess {_idx}")
             process = get_subprocess(config=self.config, target=self.target, sockets=self.sockets)
+            logger.debug(f"Starting subprocess {_idx} [{process.pid}]")
             process.start()
+            logger.debug(f"Process started and now getting appended {_idx} [{process.pid}]")
             self.processes.append(process)
```

Comment:
@vschettino-asimov and @socratesx-asimov : [Based on the logs I added to the notebook](https://app.datadoghq.com/s/8ed6dbd2-9e4b-11ec-854a-da7ad0900002/7m3-fru-84b), you can see that process.start() is running to completion. But the process' `target` is never getting called. `target` is the `Server.run` call that servers the fastapi app from the uvicorn workers.